### PR TITLE
Add infra and indexing sig

### PR DESCRIPTION
**Pull request by robcxyz**
Please add suggestions

### CODE REVIEW

**Comment by han-so1omon** [comment]
@robcxyz Please rename the folder as `special-interest-groups/infrastructure`. I don't typically like abbreviations

**Comment by robcxyz** [comment]
@han-so1omon Done. Any other suggestions?

**Review by CyrusVorwald — APPROVED** [review]

File: special-interest-groups/infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+- [icon-tracker](https://github.com/sudoblockio/icon-tracker)
+
+## Contributing
+
+Please file issues in the [icon-tracker](https://github.com/sudoblockio/icon-tracker) or any of the relevant associated repositories. 
+
+## Contact
+
+Rob Cannon (rob@sudoblock.io)
```
Comment (han-so1omon) [inline special-interest-groups/infrastructure/README.md:22]: Should include the tracker_dev discord channel, and possibly rename that channel to infrastructure_dev

File: special-interest-groups/infrastructure/charter.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+These trackers are running in 2 regions with both dev and prod environments allowing developers to push changes to the appropriate cluster and see changes before they are ultimately pushed up into production. This SIG oversees these environments along with coordinating long term changes to the tracker. These improvements will be done by various teams across the community with this SIG serving as the planning point for implementing these changes. 
+
+## Governance
+
+This SIG adheres to the [SIG governance guidelines](/guidelines/governance/sig-governance-guidelines.md). Regular meetings will be planned on a monthly basis to engage with the community for planning / feedback purposes. 
+
+## Contacts
+
+SIG chair & technical lead: [Rob Cannon](https://github.com/robcxyz)
```
Comment (han-so1omon) [inline special-interest-groups/infrastructure/charter.md:48]: Should include the tracker_dev discord channel, same as in the readme

**Review by han-so1omon — CHANGES_REQUESTED** [review]

**Comment by han-so1omon** [comment]
@robcxyz added comments about the "contact" sections in charter and readme

File: special-interest-groups/infrastructure/charter.md (hunk 2 above)
Comment (robcxyz) [reply special-interest-groups/infrastructure/charter.md:48]: @han-so1omon - discord channel is not public. Perhaps should I put an invite link that does not expire?

Comment (han-so1omon) [reply special-interest-groups/infrastructure/charter.md:48]: I would suggest to list the channel and say "raise a ticket in the discord to request access to this channel", then provide a link to the discord server

Comment (robcxyz) [reply special-interest-groups/infrastructure/charter.md:50]: @han-so1omon - Added your suggestion and refined the projects section. 

Also added a meetings directory where the agenda can be made public. Will raise separate PR for that and would love a review then. Hoping initial meeting agenda can be template for subsequent meetings.

**Review by han-so1omon — APPROVED** [review]

**Pull request merged**